Bottom: 3085b851b213d025aba6499e79497d7f424f05a4
Top:    ae57f4cce20ccc5a3c7ce2d9c777b6815b3ab8bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:17:44 +0100

hid/gtk: Remove unused colour allocation for "BlackPixel"


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 9fd146e..76ac977 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -222,7 +222,7 @@ GhidGui _ghidgui, *ghidgui = NULL;
 
 GHidPort ghid_port, *gport;
 
-static GdkColor WhitePixel, BlackPixel;
+static GdkColor WhitePixel;
 
 static gchar		*bg_image_file;
 
@@ -2502,7 +2502,6 @@ ghid_create_pcb_widgets (void)
   GError	*err = NULL;
 
   gdk_color_parse ("white", &WhitePixel);
-  gdk_color_parse ("black", &BlackPixel);
 
   if (bg_image_file)
     ghidgui->bg_pixbuf = gdk_pixbuf_new_from_file(bg_image_file, &err);
